ftplugin/gitcommit.vim

```diff
@@ -13,7 +13,11 @@ let b:did_ftplugin = 1
 
 setlocal comments=:# commentstring=#\ %s
 setlocal nomodeline tabstop=8 formatoptions+=tl textwidth=72
-setlocal formatoptions-=c formatoptions-=r formatoptions-=o formatoptions-=q
+setlocal formatoptions-=c formatoptions-=r formatoptions-=o formatoptions-=q formatoptions+=n
+
+" Ensure hanging indents work for bulleted lists
+setlocal formatlistpat+=\\\|^\\s*[-*+]\\s\\+
```

Comment:
Was a little unsure if it's better just to set `formatlistpat` explicitly rather than appending. Happy to adjust.